common/params.cc

```diff
@@ -179,7 +179,6 @@ std::unordered_map<std::string, uint32_t> keys = {
     {"UpdaterNewReleaseNotes", CLEAR_ON_MANAGER_START},
     {"Version", PERSISTENT},
     {"VisionRadarToggle", PERSISTENT},
-    {"WideCameraOnly", PERSISTENT},
```

Comment:
This param really means "single camera mode", which we still want to support

Reply:
We want to support this on a C3 or for devices that only have 1 camera? Like webcam and sim? If the latter I think we should set it up differently

Reply:
we mostly care about it for the latter, but I don't see why C3 should be treated differently